swarmlib/__main__.py

```diff
@@ -72,6 +76,10 @@ def run_swarm():
     args = vars(parser.parse_args())
     if args:
         algorithm = args.pop('func')
+        loglevel = args.pop('log_level', "INFO")
+        level = getattr(logging, loglevel.upper(), None)
+        if level:
```

Comment:
This check should be unnecessary when the argument is a _choice_ with a default value